environment.yml

```diff
@@ -9,7 +9,6 @@ dependencies:
   - html5lib=0.9999999=py35_0
   - markdown=2.6.9=py35_0
   - protobuf=3.4.1=py35he6b9134_0
-  - tensorflow-gpu=1.8.0
```

Comment:
Not at all, I changed it with the purpose of accelerating the training, since the GPU is new.

And the name is of no necessity neither

Reply:
> Hi @sun-te Is this necessary: change of env name & supressing tensorflow package ?

@kalifou  Not at all, I changed the tensorflow version with only purpose of accelerating the training, since the GPU is new.

And the name is of no necessity neither